CONTRIBUTING.md

```diff
@@ -1,6 +1,11 @@
-# Contributing to FMRIPREP
+# Contributing to fMRIPrep
 
-Welcome to the FMRIPREP repository! We're excited you're here and want to contribute.  
+Welcome to the fMRIPrep repository! We're excited you're here and want to contribute.
+But before getting into the details of how to contribute, it is important that we establish
```

Comment:
This is a tone choice, but personally I'd rather the messaging start off as welcoming as possible, before building into specifics (it's great to have those in a pull request template ! maybe an issue template, too ?) The sentence added here feels like it's keeping people at an arm's length, which doesn't match my experience with (how I've seen you) encouraging contributions !

Reply:
Sorry, we are working on a more detailed document about this, and left that where it shouldn't. Good catch!